src/Rasterization/SVGRasterizer.php

```diff
@@ -243,7 +243,7 @@ public function getHeight()
     public function getScaleX()
     {
         if (!empty($this->viewBox)) {
-            return $this->getDocumentWidth() / $this->viewBox[2];
+            return $this->width / $this->viewBox[2];
```

Comment:
The document width and the rasterizer width are two very different things. Changing this doesn't seem like a good idea.